### PR TITLE
Emote Modifiers

### PR DESCRIPTION
**Pull request by deboralbarros**
PR adds emote modifiers to rotate an emote to left and right:

image

image

### CODE REVIEW

File: src/modules/chat/index.js [hunk 1]
```diff
@@ -29,10 +29,14 @@ const EMOTE_MODIFIERS = {
   'v!': 'bttv-emote-modifier-flip-vertical',
   'z!': 'bttv-emote-modifier-zero-space',
   'c!': 'bttv-emote-modifier-cursed',
+  'l!': 'bttv-emote-modifier-rotate-left',
+  'r!': 'bttv-emote-modifier-rotate-right',
   ffzW: 'bttv-emote-modifier-wide',
   ffzX: 'bttv-emote-modifier-flip-horizontal',
   ffzY: 'bttv-emote-modifier-flip-vertical',
   ffzCursed: 'bttv-emote-modifier-cursed',
+  ffzL: 'bttv-emote-modifier-rotate-left',
```
Comment (dclstn) [inline src/modules/chat/index.js:38]: Does frankerfacez support these modifiers? If not we should exclude `ffzL` and `ffzR`.

File: src/modules/chat/style.css [hunk 2]
```diff
@@ -33,6 +33,14 @@
   transform: scaleY(-1);
 }
 
+.bttv-emote-modifier-rotate-left img {
+  transform: rotate(-0.25turn);
```
Comment (dclstn) [inline src/modules/chat/style.css:37]: nit: `rotate(-90deg)`

File: src/modules/chat/index.js (hunk 1 above)
Comment (deboralbarros) [reply src/modules/chat/index.js]: done

File: src/modules/chat/style.css (hunk 2 above)
Comment (deboralbarros) [reply src/modules/chat/style.css]: done

File: src/modules/settings/components/settings/global/Emotes.jsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ function EmotesModule() {
               {formatMessage(
                 {
                   defaultMessage:
-                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>',
+                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>, Rotate to Left: <code>l! emoteName</code>, Rotate to Right: <code>r! emoteName</code>',
```
Comment (dclstn) [inline src/modules/settings/components/settings/global/Emotes.jsx:88]: Other changes seem good, think we could drop the "to" and just say Rotate Left and Rotate Right

Comment (deboralbarros) [reply src/modules/settings/components/settings/global/Emotes.jsx]: done

File: src/modules/chat/style.css [hunk 4]
```diff
@@ -33,6 +33,14 @@
   transform: scaleY(-1);
 }
 
+.bttv-emote-modifier-rotate-left img {
```
Comment (dclstn) [inline src/modules/chat/style.css:36]: Screenshot 2023-05-13 at 16 32 18

Overlapping isn't ideal, can we add a max-width when applying this style?

e.g. `max-width: 28px;`

Comment (deboralbarros) [reply src/modules/chat/style.css:36]: added

**Pull request merged**